prisma/migrations/20231128184811_init_0/migration.sql

```diff
@@ -0,0 +1,29 @@
+-- CreateTable
+CREATE TABLE "Role" (
```

Comment:
Not a change request but we can talk more about separate role tables with you later.